### DISCUSSION TITLE
Change settings disposition

### DISCUSSION THREAD

**Discussion by Alil2096** [supermerill/SuperSlicer#2316] [General]
Hi everyone, 
 know that SuperSlicer is a port of PrusaSlicer, but I would like to understand if there was the possibility to simplify and make the positioning of the settings more efficient.
I give an example: you now have three tabs, one for print settings, one for filament settings and one for printer settings. Because for example, it would be a lot to be able to set the retraction settings according to the filament .. or the print speeds. That is, I create a "PETG" profile and inside I have both the speed and retraction settings, which are associated with the filament and are never the same for all materials. Why are they put there fixed? So every time I have to change them or create different printer profiles. All of this is unnerving

Thanks, 
Andrea

**Comment by supermerill** [comment]
you can edit the ui files (resources/ui_layout) as you wish.
there is a readme on how to write it.
The only requirement is that print settings can't be written in another file than print.ui, same for the other ones.

mmm , I think you asked for something else but I quite don't understand what.
